### PR TITLE
add reset.css

### PR DESCRIPTION
**Pull request by Voldyzzz**

### CODE REVIEW

File: css/reset.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
```
Comment (fordelord) [inline css/reset.css]: I think we do not need this comment in this file. Can you remove this?

File: css/reset.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
```
Comment (fordelord) [inline css/reset.css:90]: I think we need to add there the space to be more consistent

**Comment by ntorbinskiy** [comment]
@Voldyzzz I think you should connect this file in `index.html` file, because now this file doesn't do anything (it lives its own life😉)

**Comment by ntorbinskiy** [comment]
All works pretty well, merging it, but:

1. in future try to name branch like similar to task name, in lower case and spaces changed to dash, for example: 
Create the result education section --> create-the-result-education-section

2. Start all commits with capital letter

**Pull request merged**